### ISSUE TITLE
Documentation missing for simhyd

### ISSUE THREAD

**Issue by josephguillaume** [josephguillaume/hydromad#37]

**Comment by josephguillaume** [comment]
@WillemVervoort - are you happy to have a go at this one, perhaps after the Roxygen2 conversion?

**Comment by WillemVervoort** [comment]
Yes, I will tackle this after the README is finished (this arvo)

**Comment by josephguillaume** [comment]
Now ready when you are on this one. Let me know if you want to discuss anything.

**Comment by WillemVervoort** [comment]
OK, "this arvo" has come and gone, but this is next on my list
